package.json

```diff
@@ -110,13 +111,17 @@
     "build-themes": "gulp style-compiler-themes",
     "build-themebuilder-assets": "gulp style-compiler-tb-assets",
     "dev": "gulp dev",
-    "test-env": "node testing/launch"
+    "test-env": "node testing/launch",
+    "generate-dts": "node ./build/generate-dts.js"
   },
   "browserslist": [
     "last 2 versions",
     "ios > 8",
     "ie > 10",
     "> 1%"
   ],
-  "pre-commit": "lint-staged"
+  "pre-commit": [
+    "lint-staged",
+    "generate-dts"
```

Comment:
How do you think about renaming it to `update-ts` ? (as well as the global command)